server/src/main/java/io/crate/planner/statement/CopyPlan.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.planner.statement;
+
+import static io.crate.data.SentinelRow.SENTINEL;
+
+import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
+
+import io.crate.data.CollectingRowConsumer;
+import io.crate.data.InMemoryBatchIterator;
+import io.crate.data.Row;
+import io.crate.data.Row1;
+import io.crate.data.RowConsumer;
+import io.crate.metadata.TransactionContext;
+
+
+public abstract class CopyPlan {
+
+    public static void executeWithWaitCondition(RowConsumer consumer,
```

Comment:
Same here, please avoid including argument description inside the method name if not really needed.
```suggestion
    public static void execute(RowConsumer consumer,
```